course/assets/css/videojs_player.scss

```
@@ -12,8 +17,14 @@
 
   .vjs-control-bar {
     height: 4.4em;
-    background-color: #2b333f;
+    background-color: $control-item-bg-color;
+    color: $control-item-color;
     transition: visibility 2s, opacity 0.1s;
+    bottom: auto;
+  }
```

@MAbdurrehman12 I was reading about videojs today hoping to have some insight re  (which I don't have much yet). But regarding
> However, I'm looking into it, as to how we can show the controls even when the video has not played yet.

it seems that

```suggestion
    display: flex;
  }
```
works. Basically, just overriding the `display:none` specified by `.video-js .video-js `. Doesn't feel great messing with their CSS like this, but I guess it's what they recommend https://videojs.com/guides/skins/#creating-a-skin

---

Thank you @ChristopherChudzicki for looking into this. 

> Doesn't feel great messing with their CSS like this

I agree with this so I also tried to find any configuration for it but couldn't find any, hence I think, for now, we can override the default CSS (which we have already done at a few places).

However, the customization of the controls is still a blocker. 1 solution is to replace `video.js controls` with `youtube.js controls` and if we do that, then I think all these design updates of controls, including the one you suggested, will not be used, but I'll update this PR for now.

Thank you for this suggestion!